microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -525,8 +527,20 @@ _new as of v106_
 Whether to hide or display code that's in the ignore list, in the **Sources** tool's tree view, such as the **Page** tab.
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary v106.
-*  This checkbox is not present in Microsoft Edge Stable v104.
+*  This checkbox is present in Microsoft Edge Canary v107.
+*  This checkbox is not present in Microsoft Edge Stable v105.
+
+
+<!-- ====================================================================== -->
+## Highlight important DOM properties in the Object Properties viewer
+
+_new as of v107; added to Stable v105 through Canary v107 when v107 was released_
```

Comment:
@captainbrosset - checkbox seems as if added in all channels at once.  pls review

Reply:
I don't know when it was added, but you're right, I see it in 105, and it appears to be working there. So we should probably just say that this was added in 105.

Reply:
done